packages/@sanity/form-builder/src/inputs/ArrayInput/item/ArrayInputListItem.css

```diff
@@ -13,12 +13,16 @@
   background: var(--component-bg);
   border-radius: var(--border-radius-small);
   padding: calc(var(--extra-small-padding) - 1px);
-  transition: box-shadow 100ms;
+  transition: box-shadow 500ms;
```

Comment:
This is a nice touch! I think a duration of `250ms` would be a tad more subtle (in the long run).